bndtools.m2e/src/bndtools/m2e/MavenDependenciesRepository.java

```diff
@@ -43,9 +49,21 @@ public String getName() {
 		}
 
 		try {
-			MavenBndrunContainer mavenBndrunContainer = MavenBndrunContainer.getBndrunContainer(projectFacade, null, true,
-				true, allScopes, monitor);
-			mavenBndrunContainer.resolve()
+			MavenBndrunContainer mavenBndrunContainer = MavenBndrunContainer.getBndrunContainer(projectFacade, null,
+				true, true, allScopes, monitor);
+			MapStream.of(mavenBndrunContainer.resolve()
+				.entrySet())
```

Comment:
You don't need to say `.entrySet()`, MapStream.of also takes a map.

Reply:
fixed